Sources/ArcGISToolkit/Documentation.docc/Resources/FlyoverSceneView/FlyoverSceneViewStep3.swift

```diff
@@ -0,0 +1,35 @@
+import SwiftUI
+import ArcGIS
+import ArcGISToolkit
+
+struct FlyoverExampleView: View {
+    @State private var scene = Scene(
+        item: PortalItem(
+            portal: .arcGISOnline(connection: .anonymous),
+            id: PortalItem.ID("7558ee942b2547019f66885c44d4f0b1")!
+        )
+    )
+    
+    var body: some View {
+        FlyoverSceneView(
+            initialLocation: Point(
+                x: 4.4777,
+                y: 51.9244,
+                z: 1_000,
+                spatialReference: .wgs84
+            ),
+            translationFactor: 1_000
+        ) { sceneViewProxy in
+            SceneView(scene: scene)
+                .onSingleTapGesture { screen, _ in
+                    Task.detached {
+                        let results = try await sceneViewProxy.identifyLayers(
+                            screenPoint: screen,
+                            tolerance: 20
+                        )
+                        print("\(results.count) identify result(s).")
+                    }
```

Comment:
If we decide to keep the identity in the tutorial, we should do this in a task view modifier to display good habits when using async code. When using a task view modifier, the cancellation of the task is handled out of the box which is a good benefit. 

Also, we shouldn't put print statements in tutorial code, I would think. It would be better to have a comment like "Do something with the identified results".